global.json

```diff
@@ -26,7 +26,8 @@
         "$(MicrosoftNETCoreApp80Version)",
         "$(MicrosoftNETCoreApp90Version)"
       ]
-    }
+    },
+    "xcopy-msbuild": "17.8.3"
```

Comment:
That version does not exist. The only `17.8.*` version that is available is `17.8.5`: https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-eng/NuGet/RoslynTools.MSBuild/versions/17.8.5